ChatBot/ChatBot/DTO/RequestBodyDTO.swift

```diff
@@ -0,0 +1,14 @@
+extension OpenAI.Chat {
+    struct RequestBodyDTO: Encodable  {
```

Comment:
전체적으로 nested type이 아래처럼 두개의 공백을 두시고 계신데 어떤 의미가 있거나 팀 내에서 약속하신 걸까요?
```suggestion
    struct RequestBodyDTO: Encodable {
```

Reply:
엇 그냥 오타인 것 같습니다.. 수정해서 반영해두겠습니다 !